infra/modules/providers/azure/data-factory/variables.tf

```diff
@@ -68,34 +68,4 @@ variable "data_factory_trigger_frequency" {
   description = "The trigger freqency. Valid values include Minute, Hour, Day, Week, Month. Defaults to Minute."
   type        = string
   default     = "Minute"
-}
-
-variable "data_factory_dataset_sql_name" {
```

Comment:
It looks like the interface to this module is changing. Why? Is that needed for this PR?